src/src/Mastermind.java

```diff
@@ -1,16 +1,42 @@
+import java.util.ArrayList;
 
 class Mastermind {
+	private ArrayList<Attempt> attempts = new ArrayList<Attempt>();
+	private SecretCombination secretCombination;
+	private static final int NUM_ATTEMPS = 10;
```

Comment:
esta propiedad no tiene que ser estática
```suggestion
	private final int NUM_ATTEMPS = 10;
```